### PR TITLE
Kpmp 312 normalize users

### PR DESCRIPTION
**Pull request by zwright**

### CODE REVIEW

File: src/components/Upload/Forms/v1StyleFormValidator.test.js [hunk 1]
```diff
@@ -2,93 +2,90 @@ import { validate, validateNotEmpty, validateDate } from './v1StyleFormValidator
...
 	});
 	it("should return 'Required' in errors.institution when no institution supplied", () => {
-		let values = { "institution": '' };
+		values.institution = "";
 		let result = validate(values);
 		expect(result.institution).toEqual('Required');
 	});
 	it("should return undefined in errors.institution when a institution supplied", () => {
-		let values = { "institution": 'name' };
+		values = {...values, "institution": 'name' };
```
Comment (rlreamy) [inline src/components/Upload/Forms/v1StyleFormValidator.test.js]: Why the change to pass in the values with a submitter?

Comment (zwright) [reply src/components/Upload/Forms/v1StyleFormValidator.test.js]: Because otherwise it was throwing an error for each test that the submitter was undefined. There's probably a better way to do it but it's what I came up with.
